src/persistence/serialize.h

```diff
@@ -28,23 +28,12 @@
 /// Most of those functions are unsafe unless otherwise specified
 /// Do not use them on untrusted data (e.g. check a signature first)
 
-QByteArray doubleToData(double num);
-QByteArray floatToData(float num);
-float dataToFloat(QByteArray data);
-QByteArray stringToData(QString str);
 QString dataToString(QByteArray data);
-float dataToRangedSingle(float min, float max, int numberOfBits, QByteArray data);
-QByteArray rangedSingleToData(float value, float min, float max, int numberOfBits);
-uint8_t dataToUint8(QByteArray data);
-uint16_t dataToUint16(QByteArray data);
-uint32_t dataToUint32(QByteArray data);
-uint64_t dataToUint64(QByteArray data);
+uint64_t dataToUint64(const QByteArray& data);
+int dataToVInt(const QByteArray& data);
```

Comment:
Just look the two following code lines below. If you're confused now, I'm totally with you… :) Btw. What does the "V" stand for? At least It is not a vector/pointer.